clients/src/main/java/org/apache/kafka/common/config/ConfigDef.java

```diff
@@ -1474,17 +1475,20 @@ public String toHtml(Map<String, String> dynamicUpdateModes) {
                 String updateMode = dynamicUpdateModes.get(key.name);
                 if (updateMode == null)
                     updateMode = "read-only";
-                addConfigDetail(b, "Update Mode", updateMode);
+                addConfigDetail(b, "Update Mode:", updateMode);
```

Comment:
No need for the extra `:`, `addConfigDetail()` already inserts it

Reply:
Done